data-access/db/prisma/schema.prisma

```diff
@@ -63,6 +63,7 @@ model User {
   stripePriceId          String?   @map(name: "stripe_price_id")
   stripeCurrentPeriodEnd DateTime? @map(name: "stripe_current_period_end")
 
+  UserProfile UserProfile?
```

Comment:
might have to make a default

Reply:
![image](https://github.com/Ionia-Devs/chameleon/assets/103612769/9ecee0d8-fbd0-47be-a5c4-2c1f21f45adb)